pkg/cloud/ibmcloud/options/options.go

```diff
@@ -22,9 +22,11 @@ import (
 )
 
 var (
-	TokenTTL time.Duration
+	TokenTTL               time.Duration
+	WaitTransactionTimeout int
 )
 
 func init() {
 	flag.DurationVar(&TokenTTL, "token_ttl", 60*time.Minute, "TTL for kubeadm bootstrap token of the target Kubernetes cluster")
+	flag.IntVar(&WaitTransactionTimeout, "wait_transaction_timeout", 600, "Timeout in seconds for waiting IBM Cloud transaction ready")
```

Comment:
It is for all trasactions. So, `s/transaction/transactions/`

Reply:
Updated.